runtime/src/main/scala/com/ing/baker/runtime/actor/serialization/modules/TypesModule.scala

```diff
@@ -4,65 +4,39 @@ import com.google.protobuf.ByteString
 import com.ing.baker.runtime.actor.protobuf
 import com.ing.baker.runtime.actor.protobuf._
 import com.ing.baker.types
-import org.joda.time
 import org.joda.time.format.ISODateTimeFormat
-import org.joda.time.{LocalDate, LocalDateTime}
+import org.joda.time.{LocalDate, LocalDateTime, LocalTime}
 
 class TypesModule extends ProtoEventAdapterModule {
 
   private def createPrimitive(p: PrimitiveType) = protobuf.Type(Type.OneofType.Primitive(p))
 
   override def toProto(ctx: ProtoEventAdapter): PartialFunction[AnyRef, scalapb.GeneratedMessage] = {
-    case types.PrimitiveType(clazz) if clazz == classOf[java.lang.Boolean] =>
-      createPrimitive(PrimitiveType.BOOLEAN_PRIMITIVE)
-    case types.PrimitiveType(clazz) if clazz == java.lang.Boolean.TYPE =>
-      createPrimitive(PrimitiveType.BOOLEAN)
-    case types.PrimitiveType(clazz) if clazz == classOf[java.lang.Byte] =>
-      createPrimitive(PrimitiveType.BYTE_PRIMITIVE)
-    case types.PrimitiveType(clazz) if clazz == java.lang.Byte.TYPE =>
+
```

Comment:
I would like there to be one more type we differentiate.
Mainly the DateTime. From a technical perspective this is no different then the Int64.
But from a functional perspective I would like to keep them seperated.
This so that other Int64 are not used as dates and dates are not used as Int64.